modules/social_features/social_activity/config/install/message.template.create_content_in_joined_group.yml

```diff
@@ -21,7 +21,7 @@ third_party_settings:
       email: email
     activity_create_direct: 0
     activity_aggregate: 0
-    activity_entity_condition: ''
+    activity_entity_condition: node_published
```

Comment:
Hi @navneet0693, don't we need an update hook here?

Reply:
@ronaldtebrake yes, I will add it. 🙈